marklogic-data-hub/build.gradle

```diff
@@ -200,6 +200,10 @@ task generateEntitySearchInterface(type: com.marklogic.client.tools.gradle.Endpo
     serviceDeclarationFile = dataServicesPath + "/entitySearch/service.json"
 }
 
+task generateFlowInterface(type: com.marklogic.client.tools.gradle.EndpointProxiesGenTask, group: dataServicesGroup) {
```

Comment:
You don't need this, the task is generated dynamically based on your service.json file. In fact, you can delete the two task definitions above it.

Reply:
Removed all 3 tasks